src/main/java/BechData.kt

```diff
@@ -0,0 +1,94 @@
+import java.math.BigInteger
+import java.util.*
+
```

Comment:
Praise: Very nice, clean and understandable code.
Nitpick: Lack of comments (at least an author tag would have been nice)